TelCo.ColorCoder/Program.cs

```diff
@@ -1,33 +1,23 @@
 ﻿using System;
 using System.Diagnostics;
 using System.Drawing;
-
 namespace TelCo.ColorCoder
 {
     /// <summary>
     /// The 25-pair color code, originally known as even-count color code, 
-    /// is a color code used to identify individual conductors in twisted-pair 
-    /// wiring for telecommunications.
-    /// This class provides the color coding and 
-    /// mapping of pair number to color and color to pair number.
+    /// is a color code used to identify individual conductors in twisted-pair wiring for telecommunications.
+    /// This class provides the color coding and mapping of pair number to color and color to pair number.
     /// </summary>
-    class Program
+    public class Program
     {
-        /// <summary>
-        /// Array of Major colors
-        /// </summary>
-        private static Color[] colorMapMajor;
-        /// <summary>
-        /// Array of minor colors
-        /// </summary>
-        private static Color[] colorMapMinor;
+        internal static Color[] MajorColorList;
+        internal static Color[] MinorColorList;
         /// <summary>
         /// data type defined to hold the two colors of clor pair
         /// </summary>
         internal class ColorPair
```

Comment:
this Color pair definition could be moved to a different file.

Reply:
@prithamvshetty I couldn't understand why this has to moved. All declaration are in Program.cs file only right